### ISSUE TITLE
Error: Cannot find module 'lodash/each'

### ISSUE THREAD

**Issue by jellea** [mjmlio/mjml#551]
It might be me, but it seems that lodash is not specified in both the package.json:

https://github.com/mjmlio/mjml/blob/master/packages/mjml/src/index.js#L2

https://github.com/mjmlio/mjml/blob/master/packages/mjml/package.json
https://github.com/mjmlio/mjml/blob/master/package.json

```
Error: Cannot find module 'lodash/each'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/<censored>/node_modules/mjml/lib/index.js:19:13)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

**Comment by iRyusa** [comment]
Hi @jellea !

Thanks for this issue, i'll add lodash the `mjml` package for the next release. 
I'm surprised that we didn't catch that earlier, do you still use npm 2.x ?

**Comment by jellea** [comment]
Hey @iRyusa! Thanks for the quick fix! My npm is on `3.10.10`

**Comment by ngarnier** [comment]
I'm closing as @iRyusa fixed that in https://github.com/mjmlio/mjml/commit/816f06ea79c3d144ebe55196091e9cb48f270c8e. Thanks for reporting @jellea!
